### ISSUE TITLE
New-GraphObject transforms 1-element length arrays to non-arrays when passed with the values parameter

### ISSUE THREAD

**Issue by adamedx** [adamedx/autographps#85]
The `New-GraphObject` command's `Values` parameter allows the caller to specify the values for certain properties of the new object. If a specified value is an array with only one element, the property in the resulting object has the value of the element within the array, rather than the array itself. If such an is sent to Graph in a request, the result will be a bad request when Graph is expecting an array.

**To Reproduce**
When running the commands below, the output JSON is missing the square brackets (`[ ]`) for the array representing the property `emailAddresses` that was specified as an array `@($emailAddress)`:
```powershell
$emailAddress = New-GraphObject -TypeClass Complex emailAddress -Property name, address -value Home, example@pain.org
New-GraphObject contact -Property givenName, emailAddresses -value 'Test user1', @($emailAddress) | convertto-json
{
    "givenName":  "Test user1",
    "emailAddresses":  {
                           "name":  "Home",
                           "address":  "example@pain.org"
}
```
**Expected behavior**
The commands above should have had the following output with an array:
```powershell
{
    "givenName":  "Test user1",
    "emailAddresses":  [
                           {
                               "name":  "Home",
                               "address":  "example@pain.org"
                           }
                       ]
}
```
Occurs on AutoGraphPS module Version `0.13.0`

**Comment by adamedx** [comment]
This issue is caused by the fact that in PowerShell, apparently by design, functions that return an array with one element will return the element, rather than the array. A workaround is to use the `,` operator just before emitting the function's output. Not sure if this is a syntactic trick, or if PowerShell's strange behavior only applies to arrays containing elements that are not arrays.

This is [apparently known behavior](https://www.bing.com/search?q=powershell+functions+do+not+return+arrays+of+length+1&cvid=61be46547c704bd2bf561e542fce270f&FORM=ANAB01&PC=U531), but it is definitely not obvious and very difficult to debug when it causes problems.

**Comment by adamedx** [comment]
Fixed with #84.
